apps/web/playwright/reschedule.e2e.ts

```diff
@@ -416,6 +418,112 @@ test.describe("Reschedule Tests", async () => {
       }
     });
   });
+
+  test.describe("Guest availability-awareness reschedule by host", () => {
+    test("host reschedule with full non-cal.com user guests shouldn't trigger guests availability awareness", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const host = await users.create();
+      const eventType = host.eventTypes[0];
+      const nonCalUserAttendees = Array.from({ length: 10 }, (_, i) => ({
+        email: `attendee-${i}@example.com`,
+        name: `Attendee Example ${i}`,
+        timeZone: "Europe/London",
+      }));
+      const booking = await bookings.create(host.id, host.username, eventType.id, {
+        attendees: {
+          createMany: {
+            data: nonCalUserAttendees,
+          },
+        },
+      });
+
+      await page.goto(`/reschedule/${booking.uid}`);
```

Comment:
```suggestion
      await page.goto(`/reschedule/${booking.uid}`);
      await page.waitForLoadState("networkidle");
```

Please wait for page loads to complete wherever necessary, or else it causes pages to be flaky. Might work locally but when run on ci, it tends to fail sometimes.